gradle/baseline.gradle

```diff
@@ -36,6 +36,11 @@ List<String> blacklistedBaselineProjects = [
 if (!blacklistedBaselineProjects.contains(project.name)) {
     apply plugin: 'com.palantir.baseline-checkstyle'
     apply plugin: 'com.palantir.baseline-findbugs'
+    apply plugin: 'findbugs'
```

Comment:
We shouldn't add this directly. `baseline-findbugs` should add it for us automatically